modules/vorbis/audio_stream_ogg_vorbis.cpp

```diff
@@ -32,6 +32,8 @@
 
 #include "core/io/file_access.h"
 #include "core/variant/typed_array.h"
+#include "modules/vorbis/resource_importer_ogg_vorbis.cpp"
```

Comment:
Was this supposed to include the header?

Reply:
The compiler complained.

Reply:
cpp files should never be included.

Reply:
oh ha